src/components/Create/TicketingForm.tsx

```diff
@@ -65,6 +71,7 @@ export default function TicketingForm({
             label: t`Allocate reserved tokens (optional)`,
             extra: t`Automatically distribute a portion of your project's reserved tokens to other Juicebox projects or ETH wallets.`,
           }}
+          reservedRate={reservedRate}
```

Comment:
**question** is there a reason we can't get the form value here instead:

```suggestion
          reservedRate={form.getFieldValue('reserved')}
```

Reply:
yes, is the state value necessary here? if it is (cc the recent discussion in discord #frontend) we can maybe just avoid relying on the `Form` entirely to simplify things

Reply:
https://user-images.githubusercontent.com/96150256/150973885-c6a71f25-b00f-4d7d-a572-9ce672ab2dca.mp4

Without the state, the new value is not updated quickly on the UI. I've removed the other reference to the form in that component, so now it uses the state for everything except the initial load.